osu.Game/Graphics/UserInterface/ScreenTitle.cs

```diff
@@ -36,18 +37,24 @@ protected IconUsage Icon
 
         protected string Title
         {
-            set => titleText.Text = value;
+            set => TitleText.Text = value;
         }
 
         protected string Section
         {
-            set => pageText.Text = value;
+            set => PageText.Text = value;
         }
 
         public Color4 AccentColour
         {
-            get => pageText.Colour;
-            set => pageText.Colour = value;
+            get => PageText.Colour;
+            set => PageText.Colour = value;
+        }
+
+        public Color4 SeparatorColour
+        {
+            get => Separator.Colour;
+            set => Separator.Colour = value;
         }
```

Comment:
Do these need to be a thing? Can't we just inject `OverlayColourProvider` locally?

Reply:
This one has to be a thing as long as `MultiHeaderTitle` exists in the current form since the multiplayer room screen doesn't have an OCP higher up in the hierarchy.

Reply:
Alright. Could be worse all things considered.